lib/resin-lint.coffee

```diff
@@ -8,14 +10,26 @@ reporter = require('coffeelint/lib/reporters/default')
 
 CONFIG_PATH = path.join(__dirname, '../config/coffeelint.json')
 
+# The linter expects the path to actual source files, for example:
+#		src/
+#		test/
+#	but depcheck expects the root of a project directory (where the
+#	package.json is). This function takes a path and propagates upwards
+#	until it contains a package.json
+getPackageJsonDir = (dir) ->
```

Comment:
The unused deps check is only used when the `-u` flag is passed right?  If so then would it make sense to have callers just do `-u ./path/to/package/json` instead?